### DISCUSSION TITLE
Possibility to adjust KFI (Consequence Class) factor for each national annex

### DISCUSSION THREAD

**Discussion by uncracked-concrete** [Dlubal-Software/Dlubal_CSharp_Client#18] [Q&A] [answered]
Is there any possiblitity to chose which KFI-factor the combination wizard should use when chosing eurocode national annexes?

As you are able to swich the load combination standard and national annex (no. 6207 for EN+CEN and no. 6034-6061 for all other national annexes) it would be nice to also be able to set the consequence class (CC1, CC2, CC3) for each national annex (can be found under "Combination Expression & Settings" menu in the software.

**Comment by Rebecca-Coding21** [comment, marked as answer]
Hi @uncracked-concrete,
the consequence class can't be changed via webservice at the moment.
